### PR TITLE
doc: update backend tech listed in README MA-834

### PR DESCRIPTION
**Pull request by e0**
Not sure why GitHub marked the line 40 change as well, could be an auto line-ending thing.

### CODE REVIEW

**Review by nanjiangshu — APPROVED** [review]

**Comment by mihai-sysbio** [comment]
This work has been planned as MA-834, so it would be good to have a look at the other items on this card before merging it in.

**Comment by e0** [comment]
@mihai-sysbio Sorry, forgot about that. Just thought that since `develop` is the default branch that is shown for the repo now it would be good if the tech matched up. Of course feel free to hold off or dismiss this PR as you wish.

**Comment by mihai-sysbio** [comment]
This was done in #543.

**Pull request closed without merging**
